src/scss/constants.scss

```diff
@@ -1,11 +1,19 @@
 $primaryColor: var(--primary-color);
 $backgroundColor: var(--background-color);
 
+$primary--lightest: #a179ff;
+$primary--light: #9669ff;
 $primary: #6f43ff;
-$dimmer-primary: #3f247d;
+$dimmer--darker: #3f247d;
```

Comment:
Wouldn't this be `$primary--dimmer` or `$primary--darker` or similar?

Reply:
Good catch.
Fixed in https://github.com/sparkletown/sparkle/pull/1374/commits/c8ed35ee719ea1855d7b0afef9ad5b619666efc1